src/Symfony/Bundle/FrameworkBundle/Command/SecretsAddCommand.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Exception\EncryptionKeyNotFoundException;
+use Symfony\Bundle\FrameworkBundle\Secret\MutableSecretStorageInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\Question;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * @author Tobias Schultze <http://tobion.de>
+ * @author Jérémy Derussé <jeremy@derusse.com>
+ */
+final class SecretsAddCommand extends Command
+{
+    protected static $defaultName = 'secrets:add';
+
+    private $secretsStorage;
+
+    public function __construct(MutableSecretStorageInterface $secretsStorage)
+    {
+        $this->secretsStorage = $secretsStorage;
+
+        parent::__construct();
+    }
+
+    protected function configure()
+    {
+        $this
+            ->setDescription('Adds a secret with the key.')
+            ->addArgument('key', InputArgument::REQUIRED)
```

Comment:
In `SecretStorageInterface` you renamed it to "name" which is fine. But it should be "name" here as well then.

Reply:
good catch (removed from `list command too)